pkg/types/cose/fuzz_test.go

```diff
@@ -17,13 +17,18 @@ package cose
 
 import (
 	"context"
+	"sync"
 	"testing"
 
+	fuzzUtils "github.com/sigstore/rekor/pkg/fuzz"
 	"github.com/sigstore/rekor/pkg/types"
 )
 
+var initter sync.Once
+
 func FuzzCreateProposedEntry(f *testing.F) {
 	f.Fuzz(func(t *testing.T, version string) {
+		initter.Do(fuzzUtils.SetFuzzLogger)
```

Comment:
should this be moved to cose v0.0.1 and populate the props with `CreateProps` like other types?